src/docs/product/cli/installation.mdx

```diff
@@ -99,4 +99,4 @@ docker run --rm -v $(pwd):/work getsentry/sentry-cli --help
 
 ## Updating and Uninstalling
 
-You can use `sentry-cli update` and `sentry-cli uninstall` to update or uninstall the sentry command line interface. These commands might be unavailable in certain situations (for instance if you install `sentry-cli` with homebrew).
+You can use `sentry-cli update` and `sentry-cli uninstall` to update or uninstall the sentry command line interface. These commands might be unavailable in certain situations, generally when `sentry-cli` has been installed by a tool like homebrew or yarn, either directly or as a dependency of another package. In those cases, the same tool will need to be used for updating and removal. If you find that `sentry-cli update` and `sentry-cli uninstall` aren't working and you don't know how the package was installed, running `which sentry-cli` will often provide a clue as to which tool to use.
```

Comment:
```suggestion
You can use `sentry-cli update` and `sentry-cli uninstall` to update or uninstall the Sentry command line interface (CLI). These commands might be unavailable in certain situations, such as when `sentry-cli` has been installed by a tool like homebrew or yarn, either directly or as a dependency of another package. In those cases, the same tool will need to be used for updating and removal. If you find that `sentry-cli update` and `sentry-cli uninstall` aren't working, and you don't know how the package was installed, running `which sentry-cli` will often provide a clue as to which tool to use.
```

Reply:
Hmmm... I said "generally" in the sense of "usually," "most of the time," etc. I feel like "such as" is more "sometimes" than "most of the time." I did fix up the end of the first sentence, though.

```suggestion
You can use `sentry-cli update` and `sentry-cli uninstall` to update or uninstall the `sentry-cli` binary. These commands may be unavailable in certain situations, generally when `sentry-cli` has been installed by a tool like homebrew or yarn, either directly or as a dependency of another package. In those cases, the same tool will need to be used for updating and removal. If you find that `sentry-cli update` and `sentry-cli uninstall` aren't working and you don't know how the package was installed, running `which sentry-cli` will often provide a clue as to which tool to use.
```